multy_core/src/EOS/EOS_facade.cpp

```diff
@@ -0,0 +1,62 @@
+/* Copyright 2018 by Multy.io
+ * Licensed under Multy.io license.
+ *
+ * See LICENSE for details
+ */
+
+#include "multy_core/src/EOS/EOS_facade.h"
+
+#include "multy_core/EOS.h"
+
+#include "multy_core/src/exception.h"
+#include "multy_core/src/exception_stream.h"
+
+namespace multy_core
+{
+namespace internal
+{
+
+EOSFacade::EOSFacade()
+{
+}
+
+EOSFacade::~EOSFacade()
+{
+}
+
+HDAccountPtr EOSFacade::make_hd_account(
+        BlockchainType ,
```

Comment:
Please don't do that, if you don't use argument in a function, you should have its name commented out, like 
`/*blockchain_type*/` so it gives some context to the reader.